baseplate/secrets/fetcher.py

```diff
@@ -267,8 +270,13 @@ def main():
             os.fchmod(f.fileno(), cfg.output.mode)
 
             json.dump({
-                "vault_token": client.token,
                 "secrets": secrets,
+                "vault": {
+                    "token": client.token,
+                    "url": cfg.vault.url,
+                },
+                # this is here for backwards compatibility
+                "vault_token": client.token,
```

Comment:
I've updated the comment to explain my intent a little more. Does that make sense?